internal/pkg/template/templates/workloads/partials/cf/nlb.yml

```diff
@@ -1,4 +1,4 @@
-PublicNetworkLoadBalancer:
+NetworkLoadBalancer:
```

Comment:
Just thinking out load: is `PublicNetworkLoadBalancerV2` necessarily bad 💭 Because i feel like this is essentially what we end up doing - we are not changing it from "public" to "generic"